app/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env", "react"]
+}
```

Comment:
I'm not fully sure about this. But I remember usually we can config the babel inside the webpack.

Reply:
To do so, https://github.com/babel/babel-loader

Reply:
They do mention you can combine babelrc with webpack or not.

Reply:
Let me try it tonight

Reply:
Yes, it is able to config in webpack.config.js